<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manajemen Buku - F1D022043_Faizah</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="menus">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="daftar_buku">
       <attribute name="title">
        <string>Daftar Buku</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>0</y>
          <width>491</width>
          <height>177</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_judul">
           <property name="text">
            <string>Judul Buku       :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_pengarang">
           <property name="text">
            <string>Penulis            :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_penulis"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_tahun">
           <property name="text">
            <string>Tahun Terbit   :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_tahun"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_kategori">
           <property name="text">
            <string>Kategori          :</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_kategori"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_tempatTerbit">
           <property name="text">
            <string>Tempat Terbit :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_tempatTerbit"/>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_judul"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_pasteClipboard">
             <property name="text">
              <string>Paste</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_penerbit">
           <property name="text">
            <string>Penerbit          :</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_penerbit"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_simpan">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>180</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Simpan</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>791</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">    QHeaderView::section {
        background-color: #ececec;
    }
</string>
        </property>
        <property name="autoScrollMargin">
         <number>0</number>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="columnCount">
         <number>7</number>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>36</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>100</number>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Judul Buku</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Penulis</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tahun Terbit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kategori</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tempat Terbit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Penerbit</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="pushButton_hapus">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>430</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: red;
border-radius: 5px;
</string>
        </property>
        <property name="text">
         <string>Hapus Data</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ekspor">
       <attribute name="title">
        <string>Ekspor</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSimpan"/>
    <addaction name="separator"/>
    <addaction name="actionEkspor_ke_CSV"/>
    <addaction name="actionKeluar"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCari_Judul"/>
    <addaction name="actionHapus_Data"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_search">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Search Bar</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_cariJudul">
       <property name="placeholderText">
        <string>Cari judul...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSimpan">
   <property name="text">
    <string>Simpan</string>
   </property>
  </action>
  <action name="actionEkspor_ke_CSV">
   <property name="text">
    <string>Ekspor ke CSV</string>
   </property>
  </action>
  <action name="actionKeluar">
   <property name="text">
    <string>Keluar</string>
   </property>
  </action>
  <action name="actionCari_Judul">
   <property name="text">
    <string>Cari Judul</string>
   </property>
  </action>
  <action name="actionHapus_Data">
   <property name="text">
    <string>Hapus Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
